hdk-rust/specs/hdk-proposal.rs

```diff
@@ -0,0 +1,39 @@
+#![hdk::zome] // this attribute-like macro operates on the whole module. This is equivalent to define_zome! in the old HDK
```

Comment:
I don't think this file should make its way into the repo in this PR...

Reply:
But it's good. I'd say let's just merge it!